styles.css

```diff
@@ -127,3 +132,177 @@ body {
   opacity: 1;
   width: 100%;
 }
+
+@media (min-width: 1900px) {
+
+  .wrapper {
+    width: 85%;
+  }
+
+  .logo {
+    padding: 40px 0 40px 97px;
+  }
+
+  .logo__circle_color_blue, .logo__circle_color_dark-blue {
+    margin-left: 53px;
+  }
+
+  .menu {
+    padding-top: 0;
+    padding-right: 39px;
+  }
+
+  .menu__item {
+    font: 500 18px/21px Roboto, arial, helvetica, sans-serif;
+  }
+
+  .header {
+    margin-bottom: 10%;
+  }
+
+  .main {
+    padding: 38px 50% 0 50px;
+    background: top -30px right 6% /47% no-repeat url(img/img.svg);
+
+  }
+
+  .main__title {
+    font: bold 66px/88px 'pt_serif', arial, helvetica, sans-serif;
+    margin-bottom: 35px;
+  }
+
+  .main__description {
+    max-width: 60%;
+    font: 22px/26px Roboto, arial, helvetica, sans-serif;
+  }
+
+  .main__button {
+    font: 500 18px/21px Roboto, arial, helvetica, sans-serif;
+    padding: 17px 28px;
+    margin-left: 2px;
+    margin-bottom: 25%;
+  }
+}
+
+@media (max-width: 768px) {
+  #checkbox {
+    opacity: 0;
+  }
+
+  #checkbox:checked~label .burger::before {
+    position: relative;
+    transform: rotate(45deg);
+    top: 7px;
+  }
+
+  #checkbox:checked~label .burger::after {
+    position: relative;
+    transform: rotate(-45deg);
+    top:-4px;
+    right: 3px;
+    }
+
+  #checkbox:checked~label .burger__menu {
+    border: none;
+  }
+
+  #checkbox:checked~.menu {
+    display: block;
+    position: fixed;
+    background: white;
+    padding-top: 50%;
+    width: 100%;
+    height: 100%;
+    text-align: center;
+    z-index: 10;
+  }
+
+  #checkbox:checked~.menu .menu__item {
+    display: block;
+    font: 500 30px/35px Roboto, arial, helvetica, sans-serif;
+    padding-bottom: 24px;
+    width: auto;
+    margin-right: 0;
+  }
+
+  .wrapper {
+    width: 100%;
+    margin: auto;
+  }
+
+  .logo {
+    padding-top: 35px;
+  }
+
+  .logo__text {
+    display: none;
+  }
+
+  .logo__circle_color_dark-blue {
+    margin-left: 23px;
+  }
+
+  .logo__circle_color_blue {
+    margin-left: 23px;
+  }
+
+
+  .burger {
+    display: block;
+    position: relative;
+    margin: 27px 8px 0 0;
+    z-index: 20;
+  }
+
+  .burger__menu {
+    display: flex;
+    width: 25px;
+    padding-top: 4px;
+    border-bottom: 3px solid #0056ec;
+  }
+
+  .burger::after, .burger::before {
```

Comment:
Codacy found an issue: [Expected selector ".burger::after" to come before selector "#checkbox:checked~.burger::after" (no-descending-specificity)](https://app.codacy.com/gh/WebHeroSchool/FirstWebsait/pullRequest?prid=6278784)